### ISSUE TITLE
Cannot login to site with multiple buttons in the login form

### ISSUE THREAD

**Issue by mdotkurmi** [danieldotnl/ha-multiscrape#146]
I am trying to scrape a website that requires login and the login form has two buttons, one for 'Login' and other one for "Forget Password?'
![image](https://user-images.githubusercontent.com/14815411/185881024-d72e3997-87c4-452d-ae15-3c69020fdf95.png)

HA Multiscrape : V6.3.2

## Configuration
```
multiscrape:
- resource: 'https://xxxxx.com/studportal/Student-Home.aspx'
  scan_interval: 3600
  form_submit:
    submit_once: True
    resource: 'https://xxxxx.com/studportal/frmLogin.aspx'
    select: "#aspnetForm"
    input:
      ctl00_ContentPlaceHolder1_txtUserID_Raw: 'user'
      ctl00$ContentPlaceHolder1$txtUserID: 'user'
      ctl00_ContentPlaceHolder1_txtPassword_Raw: 'passwd'
      ctl00$ContentPlaceHolder1$txtPassword: 'passwd'
      ctl00$ContentPlaceHolder1$BnLogin: Login
      #ctl00$ContentPlaceHolder1$BnForgotPassword: ''
  sensor:
    - select: '#studInfoPanelDetailTBL > tr > td'
      name: studentname
  log_response: true
```

**Form submitted by HA Multiscrape:**
![image](https://user-images.githubusercontent.com/14815411/185881175-4a6ebeae-59ec-48a1-b889-23e536157098.png)

**Form data in browser on successful login:**
![image](https://user-images.githubusercontent.com/14815411/185881349-74e019f2-2283-4e74-b9af-6857cbd9fdea.png)

From the log response I see that form is submitted for 'Forget Password' instead of 'Login'.
This may be because HA multiscrape also submits ctl00$ContentPlaceHolder1$BnForgotPassword: '' which I think causes the form to submit for Forget password.

Can you suggest how should I configure to remove input ctl00$ContentPlaceHolder1$BnForgotPassword: '' from bring submitted.

**Comment by danieldotnl** [comment]
Duplicate of #145

**Comment by mdotkurmi** [comment]
This issue is fixed with v6.4.0 where we can remove the inputs not required in submitted form using input_filter in the config.
